packages/client/src/utils/apiErrorStateHandle.ts

```diff
@@ -0,0 +1,10 @@
+import { PayloadAction } from '@reduxjs/toolkit';
+
+const apiErrorStateHandle = (action: PayloadAction<any>): string =>
+  action.payload &&
+  typeof action.payload === 'object' &&
+  'reason' in action.payload
```

Comment:
Не проще ли использовать [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?
```suggestion
  action.payload?.reason
```

Reply:
![image](https://user-images.githubusercontent.com/54811697/230661033-0cd3d66f-b140-4fb2-82bb-b532bc4e6ad0.png)